Renombrando rutas + Refractor a ClockIN/OUT

josegomezr opened the pull request.

josegomezr committed:
Cambios *drasticos* en el URLConf

- Se renombraron un montón de rutas para no chocar.
- Se rehicieron algunas pruebas para hacer que pasen 
  con los nuevos nombres de ruta.

josegomezr committed:
:: MAJOR CHANGE ::

- Refractor al endpoint de clockin + tests

alacret commented on api/serializers/clockin_serializer.py:
```diff
@@ -14,101 +14,82 @@ class Meta:
+
+        delta = datetime.timedelta(minutes=delta_minutes)
+
+        if now > shift.ending_at + delta:
```
Pon qui un BIG TODO: para poner el cron

josegomezr committed:
Merge branch 'develop' of github.com:JobCore/api into develop

josegomezr committed:
Validación + Test para ClockIN/OUT

ClockIN/OUT ahora no permite que se haga la operación si el
employee no forma parte del shift

josegomezr committed:
DB Asserts para clockin + AutoClockOut

Para el AutoClockOut se usa la fecha `started_at` menos un segundo.
